srcs/class/response.hpp

```diff
@@ -1,13 +1,18 @@
-#pragma once
+#ifndef RESPONSE_HPP
+#	define RESPONSE_HPP
```

Comment:
Même suggestion que pour l'autre header!
```suggestion
#pragma once
```